src/plugins/vocablink.js

```diff
@@ -0,0 +1,67 @@
+let redact;
+
+const VOCABLINK_RE = /^\[v ([^\]]+)\](?:\[([^\]]+)\])?/;
+const VOCABLINK = 'vocablink';
+
+/**
+ * Plugin that adds support for Curriculum Builder's vocablinks.
+ *
+ * Note that vocab links are ONLY supported in redaction mode; rendering them
+ * out requires CurriculumBuilder database access, so they can only be rendered
+ * by Curriculum Builder itself.
+ *
+ * see https://github.com/mrjoshida/curriculumbuilder/blob/bf74aa5/curriculumBuilder/vocablinks.py
+ * @requires restorationRegistration
+ */
+module.exports = function vocablink() {
```

Comment:
just to put a comment from our offline conversation into writing, it would be nice to some day pull all of these plugin files which implement a new node type into a certain subdirectory